src/main/java/com/kodilla/ecommercee/domain/User.java

```diff
@@ -0,0 +1,40 @@
+package com.kodilla.ecommercee.domain;
+
+import lombok.*;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Data
+@Entity(name = "USER")
+public class User {
+
+    @NotNull
+    @Id
+    @GeneratedValue
+    @Column(name = "USER_ID", unique = true)
+    private long id;
+
+    @Column(name = "USER_NAME")
+    private String name;
+
+    @Column(name = "USER_STATUS")
+    private String status;
+
+    @Column(name = "USER_KEY")
+    private long userKey;
+
+    //@ManyToOne
```

Comment:
Tutaj tez raczej relacja OneToOne, chyba że się mylę to niech mnie ktoś poprawi

Reply:
Ja to zinterpretowałem, że status może być różny np. zalogowany, wylogowany więc przyjąłem, że musi być więcej relacji do różnych statusów. @pplutap może niech to rozstrzygnie :)

Reply:
aa, ja myslałam ze to jest String i po prostu login użytkownika przy logowaniu. Ale i tak dałabym tutaj OneToMany :)

Reply:
Nie powinno być relacja, to "prosta" wartość. Jest tylko long jako klucz i tylko tyle

Reply:
jeszcze tylko usuń tą linie 29 zeby nie bylo relacji tak jak podpowiada Pawel i dla mnie jest okej